scripts/addorder.js

```diff
@@ -0,0 +1,25 @@
+function show_row2() {
+  var r1_price = document.getElementById("price1").value.trim();
+  var r2 = document.getElementById("row_2");
+  if (r1_price != "") {
```

Comment:
The user could (for whatever reason) just put in the price and the next row will show up despite the other fields being blank. Not sure if this is a big deal though